### ISSUE TITLE
Is it possible to pass preselected values?

### ISSUE THREAD

**Issue by akshay-maestro** [sauzy34/react-native-multi-selectbox#81]
Is it possible to pass preselected values to SelectBox?

**Comment by AlexCernik** [comment]
Hi, you solved this problem?

**Comment by silencie** [comment]
I'm having the same issue. My plan was to use this library in production but as it seem there are no updates / bugfixes coming anymore since months.

**Comment by akshay-maestro** [comment]
I used other library 'react-native-element-dropdown' for multiselect and dropdown where I can provide preselected values.

**Comment by silencie** [comment]
> I used other library 'react-native-element-dropdown' for multiselect and dropdown where I can provide preselected values.

Thank you so much. You are a lifesaver.

**Comment by adaboubvincent** [comment]
> I used other library 'react-native-element-dropdown' for multiselect and dropdown where I can provide preselected values.

can you share the code with us? how you did it?

**Comment by sauravgupta-dotcom** [comment]
Yes — SelectBox is **controlled** in **2.0** (same idea as 1.x, now typed):

- **Single:** `value` + `onChange` (initial state can be a full `{ id, item }` option)
- **Multi:** `selectedValues` + `onMultiSelect` / `onTapClose`

Examples in the [package README](https://github.com/sauzy34/react-native-multi-selectbox/blob/master/packages/multi-selectbox/README.md) and [live demo](https://sauzy34.github.io/react-native-multi-selectbox/) (timezone / skills start preselected). Closing as completed.
